storage/storage_tests/storage_tests.cpp

```diff
@@ -1227,4 +1221,39 @@ UNIT_TEST(StorageTest_ParseStatus)
   TEST_EQUAL(StatusAndError(NodeStatus::Downloading, NodeErrorCode::NoError),
              ParseStatus(Status::EDownloading), ());
 }
+
+UNIT_TEST(StorageTest_ForEachInSubtree)
+{
+  Storage storage(kSingleMwmCountriesTxt, make_unique<TestMapFilesDownloader>());
+
+  TCountriesVec leafVec;
+  auto const forEach = [&leafVec](TCountryId const & descendantCountryId, bool expandableNode)
+  {
+    if (!expandableNode)
+      leafVec.push_back(descendantCountryId);
+  };
+  storage.ForEachInSubtree(storage.GetRootId(), forEach);
+
+  TCountriesVec const expectedLeafVec = {"Abkhazia", "Algeria_Central", "Algeria_Coast", "South Korea_South"};
+  TEST_EQUAL(leafVec, expectedLeafVec, ());
+}
+
+UNIT_TEST(StorageTest_CalcLimitRect)
+{
+  Storage storage(COUNTRIES_FILE);
```

Comment:
1. I removed an unnecessary param (COUNTRIES_FILE);
2. It's not test constructor. Storage can be created based on COUNTRIES_FILE and COUNTRIES_MIGRATE_FILE migration process;
> Please, get rid of testing constructor and create a factory method instead;
3. Yes, we can consider it, but I guess it's a question of another PR;